modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/dynamic/data/mapping/util/JournalArticleImportDDMFormFieldValueTransformer.java

```diff
@@ -68,26 +70,53 @@ public void transform(DDMFormFieldValue ddmFormFieldValue)
 
 			JournalArticle journalArticle = null;
 
+			long originalArticlePrimaryKey = jsonObject.getLong(
+				"articlePrimaryKey");
+			long originalClassPK = jsonObject.getLong("classPK");
+
 			long articlePrimaryKey = GetterUtil.getLong(
 				_portletDataContext.getNewPrimaryKey(
 					JournalArticle.class + ".primaryKey",
-					jsonObject.getLong("articlePrimaryKey")));
+					originalArticlePrimaryKey));
 
 			if (articlePrimaryKey != 0) {
 				journalArticle =
 					_journalArticleLocalService.fetchJournalArticle(
 						articlePrimaryKey);
 			}
 
+			if ((journalArticle == null) && (originalClassPK != 0)) {
+				Map<Long, Long> primaryKeys =
+					(Map<Long, Long>)_portletDataContext.getNewPrimaryKeysMap(
+						JournalArticle.class);
+
+				articlePrimaryKey = MapUtil.getLong(
+					primaryKeys, originalClassPK);
+
+				journalArticle = _journalArticleLocalService.fetchLatestArticle(
+					articlePrimaryKey);
+			}
+
 			if (journalArticle == null) {
 				if (_log.isWarnEnabled()) {
 					_log.warn(
 						"Unable to get journal article with primary key " +
 							articlePrimaryKey);
 				}
 
-				_portletDataContext.removePrimaryKey(
-					ExportImportPathUtil.getModelPath(_stagedModel));
+				if ((originalArticlePrimaryKey != 0) ||
+					(originalClassPK != 0)) {
+
+					Map<String, String> postProcessArticleUuids =
+						(Map<String, String>)
+							_portletDataContext.getNewPrimaryKeysMap(
+								JournalArticle.class +
+									".postProcessArticleUuids");
```

Comment:
For consistency, the attribute name needs to be singular (see other usages). Something like "postProcessArticleUuid", "articleUuid" or even "uuid" would be fine.
```
 _portletDataContext.getNewPrimaryKeysMap(JournalArticle.class + ".postProcessArticleUuid")
```